django/utils/cache.py

```diff
@@ -443,3 +444,35 @@ def _to_tuple(s):
     if len(t) == 2:
         return t[0].lower(), t[1]
     return t[0].lower(), True
+
+
+def invalidate_view_cache(path=None, request=None, key_prefix=None, cache=None):
+    """
+    Delete a view cache key based on either a relative URL (``path``)
+    or a request object (``request``).
```

Comment:
Does "view cache" relate to `cache_page` clearly enough? Maybe... 

```suggestion
    Delete a cached page key based on either a relative URL (``path``)
    or a request object (``request``).
```